### PR TITLE
docs: remove off:off references in the API documentation

### PR DESCRIPTION
**Pull request by albatrousse**
removal of the off:off references in the API documentation

### CODE REVIEW

File: docs/api/index.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ While testing your applications, **make all API requests to the staging environm
 
 [staging_url]: https://world.openfoodfacts.net
 
-> **Warning**: The staging environment has an extra level of [HTTP Basic Authentication][basic_auth] (username: `off`, password: `off`). When making API requests to staging, you may use [https://off:off@world.openfoodfacts.net/][staging_url] as the base URL to include the authentication.
+> **Warning**: The staging environment has an extra level of [HTTP Basic Authentication][basic_auth] (username: `off`, password: `off`). When making API requests to staging, you may use [https://@world.openfoodfacts.net/][staging_url] as the base URL to include the authentication.
```
Comment (alexgarel) [inline docs/api/index.md:61]: You have to remove all the paragraph !

File: docs/api/ref/api-v3.yml [hunk 2]
```diff
@@ -20,7 +20,7 @@ servers:
   - url: 'https://world.openfoodfacts.org'
     description: prod
   - description: dev
-    url: 'https://off:off@world.openfoodfacts.net'
+    url: 'https://@world.openfoodfacts.net'
```
Comment (alexgarel) [inline docs/api/ref/api-v3.yml:23]: you have to remove the '@' (also in the other places)

File: docs/api/tutorial-uploading-photo-to-a-product.md [hunk 3]
```diff
@@ -80,7 +80,7 @@ To add a new image for ingredients in English  to *100% Real Orange Juice*, the
 If the image is in the `images/real-orange-juice-ingredients.jpg`, we can use curl (thanks to the special '@' attributes, which enables reading from a file):
 
 ```bash
-curl -XPOST -u off:off  https://world.openfoodfacts.net/cgi/product_image_upload.pl \
+curl -XPOST -u   https://world.openfoodfacts.net/cgi/product_image_upload.pl \
```
Comment (alexgarel) [inline docs/api/tutorial-uploading-photo-to-a-product.md:83]: remove the -u also ! (see curl manpage)

**Review by alexgarel — CHANGES_REQUESTED** [review]
Hum, you did it too literally ! 

Try again ;-)

File: docs/api/ref/api-v3.yml (hunk 2 above)
Comment (alexgarel) [reply docs/api/ref/api-v3.yml:23]: https://developer.mozilla.org/fr/docs/Web/HTTP/Authentication#acc%C3%A8s_avec_identifiants_dans_lurl

**Comment by albatrousse** [comment]
Oh my bad, I just did it like a robot @.@. I will correct this right now

**Review by stephanegigandet — APPROVED** [review]
Looks good, thank you!

**Review by alexgarel — APPROVED** [review]
Thanks

**Pull request merged**